crates/store/re_log_types/src/index/time_type.rs

```diff
@@ -240,8 +240,8 @@ mod tests {
     fn test_format_parse() {
         let cases = [
             (TimeInt::STATIC, "<static>"),
-            (TimeInt::MIN, "−∞"),
-            (TimeInt::MAX, "+∞"),
+            (TimeInt::MIN, "beginning"),
+            (TimeInt::MAX, "end"),
```

Comment:
but we can still parse the old thing, so kinda strange that it falls out of the test here

Reply:
Hmm maybe, the test checks both that the value formatted equals the given string. And that parsing the given string equals the value though.